File: 351003/Pulish/discussion/dto.py
from pydantic import BaseModel, Field


class CommentRequestTo(BaseModel):
    id: int | None = None
    content: str = Field(min_length=2, max_length=2048)
    topicId: int


class CommentResponseTo(BaseModel):
    id: int
    content: str
    topicId: int

File: 351002/Filipenko/services/discussion-service/internal/entity/dto.go
package entity

type CommentRequestTo struct {
	ID      int64  `json:"id,omitempty"`
	TopicID int64  `json:"topicId"`
	Country string `json:"country"`
	Content string `json:"content"`
}

type CommentResponseTo struct {
	ID      int64  `json:"id"`
	TopicID int64  `json:"topicId"`
	Country string `json:"country"`
	Content string `json:"content"`
	State   string `json:"state"`
}

type KafkaCommentMessage struct {
	ID            int64  `json:"id"`
	TopicID       int64  `json:"topicId"`
	Country       string `json:"country"`
	Content       string `json:"content"`
	State         string `json:"state"`
	Operation     string `json:"operation"`
	CorrelationID string `json:"correlationId"`
}
